charts/schema-registry/Chart.yaml

```diff
@@ -12,6 +12,6 @@ apiVersion: v2
 name: schema-registry
 description: Schema Registry Helm chart for Kubernetes
 type: application
-version: 0.0.1
-appVersion: 0.0.1
+version: 0.1.0
```

Comment:
if i understand this correctly, this is about the schema registry version.. right? 
since we are adding the auth string changes in 0.1.1-SNAPSHOT and 0.2.0-SNAPSHOT branches, so should this be updated accordingly?

basically without the auth string changes (#145) we would not really be able to verify the usage of init containers for new version of key cloak based auth handler.

Reply:
I think the 2 issues are separate.    Versioning, aligning with the auth string changes etc is one.

The other is the generic ability to have the schema registry support init containers.   These initcontainers for all intents and purposes  would _likely_ be used specifically for copying in jar such as implementations of the [AuthHandler](https://github.com/pravega/schema-registry/blob/master/server/src/main/java/io/pravega/schemaregistry/server/rest/auth/AuthHandlerManager.java#L44), but in theory, they could be used for other things.

Also they could be used to copy in arbitrary implementations of the AuthHandler interface, not just Keycloak based ones.

The code in this PR would be validated by using any docker image containing some file, and testing that by configuring the right options in the values.yaml file of this chart, the file shows up in the main container.     That is really the generic essence of this current PR.